lib/point.js

```diff
@@ -182,9 +183,10 @@ Point.prototype.multiplyTwo = function (j, x, k) {
   return R
 }
 
+var ZERO = Buffer.from('00', 'hex')
 Point.prototype.getEncoded = function (compressed) {
   if (compressed == null) compressed = this.compressed
-  if (this.curve.isInfinity(this)) return new Buffer('00', 'hex') // Infinity point encoded is simply '00'
+  if (this.curve.isInfinity(this)) return ZERO // Infinity point encoded is simply '00'
```

Comment:
@fanatid everything could be considered a mutable object... I don't think we need to?